k8s/sonarqube/README.md

```diff
@@ -387,23 +383,21 @@ Below shell script will restore data from `postgresql/backup.sql` to PostgreSQL
       -- psql -U postgres
     ```
 
-1. Next copy files from current folder to folder `$SONARQUBE_HOME/data` in SonarQube application pod:
+1. (Optional) Use the following command to copy data files from your local folder to `$SONARQUBE_HOME/data` in the SonarQube Pod:
```

Comment:
Why this step started to be optional?

Reply:
I don't remember clearly, but it might have been a typo.

```suggestion
1. Use the following command to copy data files from your local folder to `$SONARQUBE_HOME/data` in the SonarQube Pod:
```